### PR TITLE
Fix #305428 - TextLine doesn't cause a multi-measure break

### PR DESCRIPTION
**Pull request by njvdberg**
Resolves: https://musescore.org/en/node/305428

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Travis CI mtest fail, see https://travis-ci.org/github/musescore/MuseScore/jobs/686958958#L4302-L4328

File: libmscore/layout.cpp [hunk 1]
```diff
@@ -2172,7 +2172,7 @@ static bool breakMultiMeasureRest(Measure* m)
       for (auto i : sl) {
             Spanner* s = i.value;
             // break for first measure of volta and first measure *after* volta
```
Comment (Spire42) [inline libmscore/layout.cpp:2174]: Shouldn't this comment be updated?

Comment (njvdberg) [reply libmscore/layout.cpp]: Better, indeed. Done.

**Comment by njvdberg** [comment]
Corrected the reference, mtest is now running fine locally.

**Pull request merged**